x/mobileproxy/clib/clib.go

```diff
@@ -0,0 +1,63 @@
+package clib
+
+// #include <stdlib.h>
+import (
+	"C"
+	"runtime/cgo"
+
+	"github.com/Jigsaw-Code/outline-sdk/x/mobileproxy"
+)
+
+type ProxyHandle = cgo.Handle
+type StreamDialerHandle = cgo.Handle
+
+//export NewStreamDialerFromConfig
+func NewStreamDialerFromConfig(config *C.char) StreamDialerHandle {
```

Comment:
`StreamDialerHandle` should be defined as `unsafe.Pointer` as well? It would be helpful if you could paste the generated `.h` file into the PR description, so we can get a better idea of how these C functions looks like.

Reply:
Should be with the binary file (libxxx.so or xxx.dll).